tests/ui/suggestions/issue-68049-2.stderr

```diff
@@ -6,8 +6,8 @@ LL |       *input = self.0;
    |
 help: consider changing that to be a mutable reference
    |
-LL |   fn example(&self, input: &mut i32); // should suggest here
-   |                            ~~~~~~~~
```

Comment:
I'm going to leave this for a follow-up PR if that's OK with you?

Reply:
As it is causing a clear visual regression I would prefer to investigate further and try to land the whole fix in a single PR (to avoid risking this slipping into beta by accident simply because we forgot to follow up).

Reply:
This is now fixed, right?